index.html

```diff
@@ -16,7 +16,7 @@
 </head>
 <body>
     <div id="catsscore">
-        <div class="container">
+        <div class="container-fluid">
```

Comment:
I am not sure about this commit in general.
If the original system does not expand to full screen width, then our skin probably should not either.
At least, we should discuss this.